desktop/apps/artwork/components/additional_info/index.styl

```diff
@@ -1,8 +1,11 @@
 @require '../../../../components/side_tabs'
 
 .artwork-additional-info
-  //
+
   &__content
+    p
+      margin-bottom 1em
```

Comment:
This margin was taken from https://www.artsy.net/article/artsy-editorial-women-bauhaus-school

![screen shot 2017-06-12 at 1 48 04 pm](https://user-images.githubusercontent.com/236943/27054473-d7e7a5c4-4f75-11e7-96d6-085de9110da3.png)